raiden/transfer/channel.py

```diff
@@ -1722,11 +1724,32 @@ def handle_channel_closed(
                 channel_state.token_network_identifier,
                 balance_proof,
             )
+            channel_state.update_transaction = TransactionExecutionStatus(
+                started_block_number=state_change.closed_block_number,
+                finished_block_number=None,
+                result=None,
+            )
             events.append(update)
 
     return TransitionResult(channel_state, events)
 
 
+def handle_channel_updated_transfer(
+        channel_state: NettingChannelState,
+        state_change: ContractReceiveUpdateTransfer,
+        block_number: typing.BlockNumber,
+) -> TransitionResult:
+    if state_change.channel_identifier == channel_state.identifier:
+        # update transfer was called, make sure we don't call it again
+        channel_state.update_transaction = TransactionExecutionStatus(
+            started_block_number=block_number,
+            finished_block_number=block_number,
```

Comment:
humm, this should probably be removed, we are not using the event block number in the state change, but the chainstate's block number, which may be significantly behind during a restart.

but this is for another issue/PR